src/patternfly/components/Wizard/wizard.scss

```diff
@@ -13,6 +13,7 @@
   }
 
   // Header
+  --pf-c-wizard__header--BackgroundColor: var(--pf-global--BackgroundColor--dark-100);
```

Comment:
do you need `dark` in the variable name, shouldn't the background color change to the dark color because of @include pf-t-dark.